### DISCUSSION TITLE
Include additional information in Signature details

### DISCUSSION THREAD

**Discussion by danilozacyac** [Keyfactor/signserver-ce#99] [Q&A]
Is there some way to include signature details for a PDF signature like:

1. Reason of the signature
2. Role of the signer

I been looking through the documentation but I can't find anything about this

**Comment by mlundblad** [comment]
Hi!

There is a REASON worker property for the PDFSigner that can be set, this will set the reason included in the signature.
This parameter is also possible to override via request parameters.

There is currently no option to set role.
